commands/web/symlink-project

```diff
@@ -8,6 +8,7 @@
 ## ExecRaw: true
 
 export _WEB_ROOT=$DDEV_DOCROOT
+export DRUPAL_PROJECTS_PATH=$DDEV_DRUPAL_CONTRIB_SYMLINK_DEST
```

Comment:
```suggestion
export DRUPAL_PROJECTS_PATH=${DRUPAL_PROJECTS_PATH:-modules/custom}
```
I think you could simplify by using a fallback. Then folks can just set DRUPAL_PROJECTS_PATH when they want to override.